src/software/new_geom/segment.h

```diff
@@ -9,27 +9,43 @@
 // TODO (Issue #1098): Refactor this to fit the new_geom hierarchy
 class Segment final
 {
-   private:
-    Point start;
-    Point end;
-
    public:
+    /**
+     * Sets the start point of the segment to o
+     *
+     * @param o new start point of segment
+     */
     void setSegStart(Point o)
```

Comment:
I think `setStartPoint` might be a slightly better name, just because `Seg` doesn't add a ton of value since this is in the `Segment` class and `setStart` doesn't seem quite descriptive enough to me (_but this last bit is quite subjective, so I'd be fine with `setStart` if you prefer it_).

Similarly for the end point and the getters.

Reply:
agreed, but out of scope

Reply:
I also have issues with some geom naming that we can address separately.

Reply:
Sounds good to me